# -*- coding: utf-8 -*-
# Generated by Django 1.10 on 2017-03-16 16:39
from __future__ import unicode_literals

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('fips', '0011_auto_20170315_0712'),
    ]

    operations = [
        migrations.AlterField(
            model_name='user',
            name='ref_code',
            field=models.CharField(default='efUAEmFAvj', max_length=10, verbose_name=b'Reference ID'),
        ),
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.ASCIIUsernameValidator()], verbose_name='username'),
        ),
    ]
